web-client/src/views/StartCase/StartCaseStep2.tsx

```diff
@@ -46,6 +47,7 @@ export const StartCaseStep2 = connect(
         <Hint>
           Don’t forget to remove or block out (redact) your personal information
           on all your documents, including any IRS notice(s).
+          {/* TODO: verify this text vs mockup */}
```

Comment:
this one too (I think I wrote this one, we should check with UX before everything's finalized)